Changelog.md

```diff
@@ -7,6 +7,8 @@ Breaking changes:
  * JSON AST: Removes members with ``null`` value from JSON output.
  * Type Checker: Disallow shifts by signed types.
  * Type Checker: Exponentiation and shifts of literals by non-literals will always use ``uint256`` or ``int256`` as a type.
+ * Type Checker: Disallow structs and arrays in memory or calldata if they contain nested mappings.
+ * Type Checker: Disallow assignments to state variables that contain nested mappings.
```

Comment:
NTS: If state variable is a user defined type that contains a mapping, assignments to its non-mapping members is permitted. What is not permitted is constructing the said user defined type and assigning this to state variable.